pkg/config/templates/emathroughput.yaml

```diff
@@ -0,0 +1,58 @@
+name: EMA Throughput Sampling
+description: |
+  Limit the throughput of traces to Honeycomb using an EMA sampler. This template
+  accepts traces from an OTel receiver, converts them to Honeycomb format, and
+  then samples them using an EMA throughput sampler before exporting them to
+  Honeycomb. Control the volume of traces by modifying the GoalThroughput value in
+  the properties section of the EMAThroughputSampler component.
+components:
+  - name: TraceGRPC_1
+    kind: TraceGRPC
```

Comment:
I am a little confused about the relationship between components here and the pkg/config/components. Should the "kind" match between them? Or am I misunderstanding how these are defined?

Reply:
You're right -- I just figured this out myself when doing some stuff for Sarah. I'll clean it up.

Reply:
OK, I've updated, and tested that the given templates generate valid configs and rules.